modules/statics/src/base.ts

```diff
@@ -761,6 +761,10 @@ export enum UnderlyingAsset {
   EQO = 'eqo',
   ESE = 'ese',
   ETA = 'eta',
+  'eth:cusdo' = 'eth:cusdo',
+  'eth:edu' = 'eth:edu',
+  'eth:tlos' = 'eth:tlos',
+  'eth:usdg' = 'eth:usdg',
```

Comment:
I think for eth is usually like this
```suggestion
  'cusdo' = 'cusdo',
  'edu' = 'edu',
  'tlos' = 'tlos',
  'usdg' = 'usdg',
```

Reply:
There's a collision between pTokens TLOS & the new Telos TLOS token, that's why for eth type tokens too from now on we will be following `eth:<token_symbol>`. It is mentioned in the ticket [COIN-2227](https://bitgoinc.atlassian.net/browse/COIN-2227)

Reply:
We are checking this on testnet first, since this is the first time we're using `eth:` prefix for eth type tokens